### ISSUE TITLE
Add limit to category and add “other”

### ISSUE THREAD

**Issue by DanRider** [nishantjainuk/SimpleWaterfall-PowerBI#33]
Would be awesome if in addition to Hide Zero/Blank there could be an option to choose “Top N + other” based on abs() value of the step. This will help remove clutter when there are numerous dimensional values and quickly identify top influencers. Great chart! Way better than the MS garbage. Keep up the great work!

**Comment by nishantjainuk** [comment]
@DanRider 
I am working on the next version of this visual and have implemented this feature? Would you like to test it for me before I submit it to Microsoft. I hope you can point our any issue or suggest enhancement to make it better. 

Let me know and I can send you a copy of the visual that you can load offline in your report to test. 

Thanks
Nishant

**Comment by nishantjainuk** [comment]
@DanRider You can download the latest version to test [here](https://github.com/nishantjainuk/SimpleWaterfall-PowerBI/blob/certification/Beta%20Version/SimpleWaterfall_dev.2.0.10.2.pbiviz)

**Comment by DanRider** [comment]
Yeah that would be awesome! Send it over and I will put it through the paces!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DanRider** [comment]
Just saw your link on GitHub

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DanRider** [comment]
I looked through all of the format menus and did not find where to restrict the dimension count to enable the "Other" bucket. I do see the yellow color assigned in the "Bar Color" section so I am certainly on the beta version.

**Comment by nishantjainuk** [comment]
@DanRider  I have only implemented it for scenario where you have 1 category and 1 measure. Can you try for this scenario? In this scenario you should see an option in "Chart Options"

![image](https://user-images.githubusercontent.com/24504997/94867504-43148f80-0439-11eb-94d3-1c7fda537e41.png)

![image](https://user-images.githubusercontent.com/24504997/94867476-342ddd00-0439-11eb-88ba-0584c7b9ca56.png)
I am planning to add this for other scenarios too but want to get your feedback before implementing it across the board

Thank for helping me out with the testing. 

Nishant

**Comment by DanRider** [comment]
Sorry for my slow response Nishant. For the single measure this works perfectly and exactly as i would expect! The reports where i have implemented your chart i am using multiple measures (Budget v Actual) to walk the change. It would be really useful for multiple measures

I also add a label for overall change if you are looking for more enhancement ideas

here is a sample with some random data
![waterfall](https://user-images.githubusercontent.com/45700711/95513306-2243c100-0988-11eb-923a-b7d077bd0ca8.PNG)

**Comment by nishantjainuk** [comment]
@DanRider thanks mate. Appreciate the feedback.

I habe already implemented the multiple measure. Can you download the dev version again (same link) and try. 

Any issues let me know 

Thanks 
Nishant

**Comment by DanRider** [comment]
Awesome! I was able to test it an functionally it works great. I found the tooltip label had a long description "defaultBreakdownStepOther1" the other tooltips Identify the sum for measure 1 and the sum for measure 2 below. 

Adding tooltip selections on the "fields" would be convenient. I try to avoid report tooltips since they run on hover instead of pageload like the default tool tips.

Thank you for your contribution to the community. This visual is top notch! 

![Label waterfall](https://user-images.githubusercontent.com/45700711/95518273-ba45a880-0990-11eb-9e45-e3570e166d39.png)
